form/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/global.css" />
+    <script src="/js/index.js" defer></script>
+    <title>Quiz App - Form</title>
+  </head>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <main class="main__form">
```

Comment:
hmm, I am not sure how to name this according to BEM. I would even consider to call it just `main` because these are the main stylings of the page.

Reply:
I've already assigned class "main", because I wanted to use it when styling for different media queries.
I changed `class="main__form"` to `class="main-form"`.